app/assets/styles/styles.css

```diff
@@ -33,194 +31,120 @@ p {
   line-height: 1.55;
 }
 
-.newsletter-date {
-  color: #fdfd56;
-  font-size: 0.8rem;
+.logo__btn {
+  display: flex;
+  justify-content: center;
+  margin-top: 0.7em;
+  margin-bottom: 0.7em;
 }
 
-.language-menu {
-  width: 100%;
-  font-size: 1.15rem;
-  font-family: 'Trajan Pro Regular', 'Times New Roman', serif;
-  text-align: right;
-  background: #1a1451;
-  margin-bottom: 0.625em;
-  padding: 0.5em 1em;
+@media only screen and (min-width: 400px) {
+  .logo__btn {
+    display: block;
+    position: absolute;
+    top: 0;
+    margin-top: 0;
+    margin-left: 1.5em;
+  }
 }
 
-.language-menu--item {
-  color: #fff;
-  padding: 0.475em;
+.menu {
+  display: flex;
+  justify-content: space-between;
+  font-size: 0.95rem;
+  font-family: 'Trajan Pro Bold', 'Times New Roman', serif;
+  background: #1a1451;
+  padding: 0.5em 1.5em;
 }
 
-.language-menu--item:hover {
+.menu__news-name {
   color: #fdfd56;
+  letter-spacing: 0.19em;
 }
 
-.primary-nav {
-	margin: 0.4em auto 0;
-	text-align: center;
-}
-
-.logo-brand {
-	color: #1a1451;
-	font-family: 'Trajan Pro Bold', 'Times New Roman', serif;
-	font-size: 2rem;
-  margin: 0;
-}
-
-.logo-description {
-	display: flex;
-  font-family: 'Trajan Pro Regular', 'Times New Roman', serif;
-  color: #606060;
-	justify-content: center;
-	margin-top: 0;
-	padding-left: 0;
-}
-
-.logo-description--text {
-	list-style: none;
-	padding: 0 0.313em;
-	font-size: 0.75rem;
+@media only screen and (min-width: 400px) {
+  .menu__news-name {
+    margin-left: 7em;
```

Comment:
Instead of applying the margin on the element, we can apply it to the entire menu. I like to think of it like placing together blocks on the page. What we want is the menu to start to the left of the logo. 

![Screenshot_2020-05-18 We are the heroes](https://user-images.githubusercontent.com/51812/82248534-b13f3680-991e-11ea-85da-515c1fbabad7.png)